README.md

```diff
@@ -1 +1,34 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+## Portfolio-setup-and-mobile-version-skeleton
+
+## Built With
+
+- HTML
+- CSS
+- Lighthouse
+- Webhint
+- Stylelint
+
+## Getting Started
+
+#### cloning project
+git clone git@github.com:collinsmezie/Portfolio-setup-and-mobile-version-skeleton.git
+
+## Getting packages and debuging with Stylelint npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x 
+
+
+##### For validation detection using Stylelint Run npx stylelint "**/*.{css,scss}" 
+
+##### from parent source directory 
+
+## Getting packages and debuging with Webhint npm init -y npminstall --save-dev hint@6.x 
+
+##### For validation detection using Webhint Run npx hint . 
+
+### Prerequisites
+- VsCode
+
+## Authors
+👤 **collins chime **
+
+- GitHub: [@collinsmezie](https://github.com/collinsmezie)
+
```

Comment:
- [ ] Optional:  please consider adding more links to your social pages like linkedin and twitter.